community.html

```diff
@@ -7,28 +7,31 @@
   <h1>MEAN.JS Community</h1>
 </section>
 <section class="row">
-  <div class="col-md-3">
+  <div class="col-md-4">
+    <h3>Support</h3>
+    <p>
+      Report issues, problems, and bugs in our <a href="https://github.com/meanjs/mean/issues">Github Issue Queue</a>.
+    </p>
+    <p>
+      Please email us with any query via <a href="mailto:support@meanjs.org">support@meanjs.org</a>.
+    </p>
+  </div>
+  <div class="col-md-4">
     <h3>Social</h3>
     <p>
       Follow us on <a href="http://twitter.com/meanjsorg">Twitter</a>, <a href="http://facebook.com/meanjs">Facebook</a>, and <a href="http://blog.meanjs.org">Tumblr</a> for updates, news, and information on the MEAN.JS stack.
     </p>
-  </div>
-  <div class="col-md-3">
-    <h3>Discussions</h3>
     <p>
-      Join us on the <a href="https://groups.google.com/forum/#!forum/meanjs">Google Group</a> or <a href="https://www.facebook.com/groups/meanjs">Facebook Group</a> to discuss features, questions, and suggestions. We also discuss the future of MEAN.JS in a dedicated <a href="https://trello.com/b/5FAPcode/mean-js">Trello Board</a>.
+      Join us on the <a href="https://groups.google.com/forum/#!forum/meanjs">Google Group</a> or <a href="https://www.facebook.com/groups/meanjs">Facebook Group</a> to discuss features, questions, and suggestions.
     </p>
-  </div>
-  <div class="col-md-3">
-    <h3>Email</h3>
     <p>
-      Please email us with any query via <a href="mailto:support@meanjs.org">support@meanjs.org</a>.
+      We also discuss the future of MEAN.JS in a dedicated <a href="https://trello.com/b/5FAPcode/mean-js">Trello Board</a>.
     </p>
   </div>
-  <div class="col-md-3">
-    <h3>Issues</h3>
+  <div class="col-md-4">
+    <h3>Education</h3>
```

Comment:
instead of promoting one item over the other, can you simply link from the educational section to _includes/docs/0.5.x/resources.html ?